stream-chat-android-client/src/main/java/io/getstream/chat/android/client/setup/state/ClientState.kt

```diff
@@ -71,22 +69,14 @@ public interface ClientState {
      */
     public val isInitialized: Boolean
 
+    /**
+     * If internet is available or not. This is not related to the connection of the SDK, it returns
+     * if internet is available in the device.
+     */
+    public val isInternetAvailable: Boolean
```

Comment:
What do you think about renaming to `isNetworkAvailable`?

Reply:
Sorry, I thought I had renamed it 🤦